### PR TITLE
Import the module

### PR DESCRIPTION
**Pull request by jhelwig**

### CODE REVIEW

File: spec/lib/puppet_spec/compiler.rb [hunk 1]
```diff
@@ -90,7 +90,7 @@ def eval_and_collect_notices(code, node = Puppet::Node.new('foonode'), topscope_
   # (Parameters given by name)
   #
   def evaluate(code: 'undef', source: nil, node: Puppet::Node.new('testnode'), variables: {})
-    source_location = caller[0]
+    source_location = caller(0..0).first
```
Comment (melissa) [inline spec/lib/puppet_spec/compiler.rb]: this is so terrible..... ruby 🤦‍♀️

**Comment by melissa** [comment]
```
[0] Melissa@bubba:~/modules/puppetlabs-sshkeys_core:(jhelwig-module-updates)$ pdk validate
pdk (INFO): Running all available validators...
pdk (INFO): Using Ruby 2.4.4
pdk (INFO): Using Puppet 5.5.1
[✔] Checking metadata syntax (metadata.json tasks/*.json).
[✔] Checking module metadata style (metadata.json).
[✔] Checking Ruby code style (**/**.rb).
info: task-metadata-lint: ./: Target does not contain any files to validate (tasks/*.json).
info: puppet-syntax: ./: Target does not contain any files to validate (**/**.pp).
info: puppet-lint: ./: Target does not contain any files to validate (**/*.pp).
[0] Melissa@bubba:~/modules/puppetlabs-sshkeys_core:(jhelwig-module-updates)$ pdk test unit
pdk (INFO): Using Ruby 2.4.4
pdk (INFO): Using Puppet 5.5.1
[✔] Preparing to run the unit tests.
[✔] Running unit tests.
  Evaluated 119 tests in 5.754713 seconds: 0 failures, 0 pending.
[✔] Cleaning up after running unit tests.
```

**Comment by melissa** [comment]
I kicked off an ad hoc run to verify the acceptance tests are good. I'll merge this once those pass

**Comment by melissa** [comment]
still waiting for resources https://jenkins-master-prod-1.delivery.puppetlabs.net/view/__experimental%20automatic/job/experimental_auto_puppetlabs-sshkeys_core_intn-sys_nightly-adhoc/1/

**Comment by melissa** [comment]
https://jenkins-master-prod-1.delivery.puppetlabs.net/view/__experimental%20automatic/job/experimental_auto_puppetlabs-sshkeys_core_intn-sys_nightly-adhoc/1/ passed!

File: spec/lib/puppet_spec/compiler.rb (hunk 1 above)
Comment (jhelwig) [reply spec/lib/puppet_spec/compiler.rb]: Yeah, when I was reading about that Rubocop lint it was just making me sad. It makes sense in a roundabout way, but it's still _horrible_ ergonomics.

**Comment by joshcooper** [comment]
I think we lost `spec/unit/provider/ssh_authorized_key/parsed_spec.rb` during the migration?

**Comment by joshcooper** [comment]
Also `spec/fixtures/unit/provider/ssh_authorized_key`

**Pull request merged**